src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -937,4 +937,16 @@
   <data name="ColorPicker_Editor.Text" xml:space="preserve">
     <value>Editor</value>
   </data>
+  <data name="FancyZones_OverlappingZonesLargest.Content" xml:space="preserve">
+    <value>Largest zone by area</value>
+  </data>
+  <data name="FancyZones_OverlappingZonesPositional.Content" xml:space="preserve">
+    <value>Select a zone based on position inside overlap</value>

```

Comment:
Let's try to improve this description to make it clear that this algorithm will always make all zones reachable.

`The overlapped area is split in sub areas`
or
`The overlapped area is split in sub areas for each zone`

or an alternative description is welcomed.

Reply:
"Select one of the overlapping zones based in window position."

Reply:
@htcfreek 
not sure if that communicates the concept that all overlapping zones will be reachable moving the mouse around the overlapped area.

We will provide a detailed explanation in the docs, but we still need to communicate the concept in the settings.

Reply:
"Select zone based on mouse hover"

I think that the general meaning is claer because of the Drop-down title.

Reply:
Sorry, I'm still not convinced it makes it clear enough that it provides a way to activate all zones splitting the overlapped area between each zone.

Reply:
This is my final proposal, we have to timebox these type of decisions otherwise we spend to much time.

```
When multiple zones overlap:
  - Activate the smallest zone by area
  - Activate the largest zone by area
  - Split the overlapped area in multiple activation targets
```